modules/oc-mirror-installing-plugin.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="installation-oc-mirror-installing-plugin_{context}"]
+= Installing the oc-mirror OpenShift CLI plug-in
+
+To use the oc-mirror OpenShift CLI plug-in to mirror registry images, you must install the plug-in. If you are mirroring image sets in a fully disconnected environment, ensure that you install the oc-mirror plug-in on all required hosts.
```

Comment:
It would be the host that has access to the internal, and then whatever host you're on in the disconnected env that has access to the mirror registry. I'll play with it to see if I can clarify